go/vt/vttablet/tabletmanager/rpc_vreplication.go

```diff
@@ -44,3 +49,73 @@ func (tm *TabletManager) VReplicationExec(ctx context.Context, query string) (*q
 func (tm *TabletManager) VReplicationWaitForPos(ctx context.Context, id int32, pos string) error {
 	return tm.VREngine.WaitForPos(ctx, id, pos)
 }
+
+// UpdateVRWorkflow updates the sidecar databases's vreplication
+// record for this tablet's vreplication workflow stream.
+// Note: the VReplication engine creates a new controller for the
+// workflow when the record is updated, so we also in effect restart
+// the workflow via the update.
+func (tm *TabletManager) UpdateVRWorkflow(ctx context.Context, req *tabletmanagerdatapb.UpdateVRWorkflowRequest) (*tabletmanagerdatapb.UpdateVRWorkflowResponse, error) {
+	query := "select id, source, cell, tablet_types from %s.vreplication where workflow = %a"
+	bindVars := map[string]*querypb.BindVariable{
+		"wf": sqltypes.StringBindVariable(req.Workflow),
+	}
+	parsed := sqlparser.BuildParsedQuery(query, sidecardb.GetIdentifier(), ":wf")
```

Comment:
I recommend `ParseAndBind()` as a cleaner, single step way to achieve the same.

Reply:
I do use that, when I don't need to replace the sidecar db identifier (remember that there is no identifier bind variable).